buildkite/src/Command/DockerImage.dhall

```diff
@@ -60,7 +60,7 @@ let ReleaseSpec =
           , branch = "\\\${BUILDKITE_BRANCH}"
           , repo = "\\\${BUILDKITE_REPO}"
           , deb_codename = DebianVersions.DebVersion.Bullseye
-          , deb_release = "\\\${MINA_DEB_RELEASE}"
+          , deb_release = "unstable"
```

Comment:
In CI we will always use unstable, others stable etc release channel are accessible only by release manager tooling upgrade